packages/transport/src/modules/files/FileController.ts

```diff
@@ -264,4 +270,12 @@ export class FileController extends TransportController {
 
         return decrypt;
     }
+
+    public async cleanupFilesOfDecomposedRelationship(peer: CoreAddress): Promise<void> {
+        const tokenDocs = await this.getFiles({ "cache.createdBy": peer.toString() });
```

Comment:
What happens with files send to this Relationship? Or with files received from this relationship which were not created by this relationship? (I don't know if the latter is actually possible?)

Reply:
- Naming: filesDocs?
- We should not clean up files so far, there still are some open questions, as we do not fetch all files with this query and might delete too many (as they are not linked to one relationship)